server/jetty/src/main/java/io/deephaven/server/jetty/JettyBackedGrpcServer.java

```diff
@@ -170,6 +171,16 @@ public JettyBackedGrpcServer(
         // Wire up the provided grpc filter
         context.addFilter(new FilterHolder(filter), "/*", EnumSet.noneOf(DispatcherType.class));
 
+        final JsPluginsZipFilesystem fs;
+        try {
+            fs = JsPlugins.initJsPlugins(Configuration.getInstance());
```

Comment:
In the latest version, I actually _am_ injecting `Configuration`.

Reply:
Right, I'm suggesting not bothering with that - more source churn, minimal payoff (since Configuration isn't mockable, can only have One True Instance, etc), and just referencing Configuration.getInstance() once where it is read

Reply:
Okay, will undo the Configuration injection.